strategies.py

```diff
@@ -175,14 +173,17 @@ def pick_move(self):
 
         Highest N is most robust indicator. In the early stage of the game, pick
         a move weighted by visit count; later on, pick the absolute max.'''
-        if self.root.position.n >= self.temp_threshold:
-            fcoord = np.argmax(self.root.child_N)
-        else:
-            cdf = self.root.child_N.cumsum()
+        if (not self.two_player_mode and
+                self.root.position.n < self.temp_threshold):
+            cdf = self.root.children_as_pi(squash=True).cumsum()
             cdf /= cdf[-2]  # Prevents passing via softpick.
             selection = random.random()
             fcoord = cdf.searchsorted(selection)
             assert self.root.child_N[fcoord] != 0
+        else:
+            # break ties in N by looking at action_score
+            fcoord = np.argmax(self.root.child_N +
+                               self.root.child_action_score / 1000)
```

Comment:
IDK what to do here anything larger than `/ 2` is likely to work in practice but  `/ (1000 + 2 * c_puct)` might be more technically correct

Reply:
i think this works fine.

Do we need to add this to C++ to keep their behaviors the same?